src/test/kotlin/no/nav/syfo/brev/narmesteleder/NarmesteLederAccessServiceSpek.kt

```diff
@@ -0,0 +1,282 @@
+                    )
+                } returns dialogmotedeltakerArbeidstaker
+            }
+
+            describe("filterExpiredDocumentsInDialogmoter") {
+                it("should return brev within overlapping oppfolgingstilfelle start date") {
+                    coEvery {
+                        oppfolgingstilfelleClientMock.oppfolgingstilfelleTokenx(
+                            any(),
+                            any(),
+                            any(),
+                            any(),
+                            any()
+                        )
+                    } returns listOf(
+                        Oppfolgingstilfelle(
+                            start = LocalDate.now().minusMonths(6),
+                            end = LocalDate.now().minusMonths(2)
+                        )
+                    )
+
+                    val moteList = runBlocking {
+                        narmesteLederAccessService.filterExpiredDocumentsInDialogmoter(
+                            moteList = dialogmoteList,
+                            narmesteLederPersonIdentNumber = UserConstants.NARMESTELEDER_FNR,
+                            arbeidstakerPersonIdentNumber = UserConstants.ARBEIDSTAKER_FNR,
+                            tokenx = anyToken,
+                            callId = anyCallId
+                        )
+                    }
+                    val brev = moteList.toNarmesteLederBrevDTOList()
+
+                    brev.size shouldBe 4
+                }
+
+                it("should only return brev created within 4 months when oppfolgingstilfelle is past 4 months") {
+                    coEvery {
+                        oppfolgingstilfelleClientMock.oppfolgingstilfelleTokenx(
+                            any(),
+                            any(),
+                            any(),
+                            any(),
+                            any()
+                        )
+                    } returns listOf(
+                        Oppfolgingstilfelle(
+                            start = LocalDate.now().minusMonths(10),
+                            end = LocalDate.now().minusMonths(6)
+                        )
+                    )
+
+                    val moteList = runBlocking {
+                        narmesteLederAccessService.filterExpiredDocumentsInDialogmoter(
+                            moteList = dialogmoteList,
+                            narmesteLederPersonIdentNumber = UserConstants.NARMESTELEDER_FNR,
+                            arbeidstakerPersonIdentNumber = UserConstants.ARBEIDSTAKER_FNR,
+                            tokenx = anyToken,
+                            callId = anyCallId
+                        )
+                    }
+                    val brev = moteList.toNarmesteLederBrevDTOList()
+
+                    brev.size shouldBe 2
+                }
+
+                it("should only return brev created within 4 months when oppfolgingstilfelle is empty list") {
```

Comment:
Også her tror jeg _within 4 months_  kan bli _in the past 4 months_. _within_ er frem i tid, altså innen 4 måneder, mens  _in the past 4 months_, kan brukes hvis du mener de siste 4 månedene frem til i dag.